src/coordinator/users.js

```diff
@@ -510,6 +519,86 @@ class Users {
       `${this._baseUrl}/users/${userId}/stacks/${stackId}`
     );
   }
+
+  /**
+   * Subscribes a user to an event
+   *
+   * API Endpoint: '/users/:userId/events/:event_id'
+   * Method: POST
+   *
+   * @param {string} userId The ID of the user
+   * @param {string} eventId The ID of the event
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtUserEvent} The newly created user event
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator.users
+   *   .subscribeEvent('36b8421a-cc4a-4204-b839-1397374fb16b', '007ca9ee-ece7-4931-9d11-9b4fd97d4d58')
+   *   .then((userEvent) => console.log(userEvent))
+   *   .catch((err) => console.log(err));
+   */
+  subscribeEvent(userId, eventId) {
+    if (!userId) {
+      return Promise.reject(
+        new Error('A user ID is required for adding a application to a user')
+      );
+    }
+
+    if (!eventId) {
+      return Promise.reject(
+        new Error('An event ID is required for adding an event to a user')
+      );
+    }
+
+    return this._request
+      .post(`${this._baseUrl}/users/${userId}/events/${eventId}`)
+      .then((response) => toCamelCase(response));
+  }
+
+  /**
+   * Removes an event subscription from a user
+   *
+   * API Endpoint: '/users/:userId/subscriptions/:user_event_subscription_id'
+   * Method: DELETE
+   *
+   * @param {string} userId The ID of the user
+   * @param {string} userEventSubscriptionId The ID of the user event subscription
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator.users
+   *   .unsubscribeEvent('36b8421a-cc4a-4204-b839-1397374fb16b', '007ca9ee-ece7-4931-9d11-9b4fd97d4d58')
```

Comment:
Would be preferable to have a correctly formed userId here too. And lose the `.then` console logging things out because it doesnt actually return anything

Reply:
👍

Reply:
👍